### ISSUE TITLE
When restoring from backup + restart the password is not displayed in properties

### ISSUE THREAD

**Issue by da-kami** [itchysats/start9-itchysats-wrapper#4]
What did I do: 

- Install itchysats package from code of https://github.com/itchysats/itchysats-wrapper/pull/1/commits/35352a2e81757584285d6938249e4dbc4506d369
- Create a backup
- Uninstall itchysats
- Then I ran into https://github.com/itchysats/start9-itchysats-wrapper/issues/3
- I restarted embassy to see if that can solve the problem
- When embassy was restarted I was able to get to the basic auth login
- When I tried to access the itchysats password in the service properties I ran into this error:

```
Docker Error YAMLError: duplicated mapping key at line 18, column 1: version: 2 ^ at generateError (file:///embassy.js:2527:12) at throwError (file:///embassy.js:2530:11) at storeMappingPair (file:///embassy.js:2642:20) at readBlockMapping (file:///embassy.js:3162:17) at composeNode (file:///embassy.js:3336:84) at readDocument (file:///embassy.js:3450:5) at loadDocuments (file:///embassy.js:3487:9) at load (file:///embassy.js:3506:23) at parse (file:///embassy.js:3516:12) at async properties (file:///embassy.js:4326:12)
```

This results in being unable to access the ItchySats interface because the password is missing. (Note: one can fetch it from the logs, those are accessible).

I could temporarily resolve this issue by uninstalling and restoring again, the password was properly configured then and I was able to access the user interface. So I think the configuration process of restoring is fine - but I then ran into https://github.com/itchysats/start9-itchysats-wrapper/issues/3 issue again.
After another restart I ran into the issue reported above again, so I ended up in an endless loop of being unable to access the interface 😅
I solved it by saving the password outside of the properies... With the DNS problem solved I was then able to access the interface. 

It has to be evaluated if this is a problem with how we setup the password configuration or if this is a problem of the underlying configuration stack when `restore` from backup + restarting.

**Comment by da-kami** [comment]
The problem of the duplicated values is that we append to `stats.yml` but the file should be replaced. 
This should be fixed on #1

**Comment by da-kami** [comment]
I can't test this at the moment but it should be solved with https://github.com/itchysats/start9-itchysats-wrapper/pull/1/commits/b584c105b4145efedf585f4e35f25f8549d599fe - I will see to test it tomorrow morning.

**Comment by da-kami** [comment]
> I can't test this at the moment but it should be solved with [b584c10](https://github.com/itchysats/start9-itchysats-wrapper/commit/b584c105b4145efedf585f4e35f25f8549d599fe) - I will see to test it tomorrow morning.

I tested it just now (package, fresh install of package as service, create backup, remove, recover from backup, restart) and the problem is resolved!

Closing this issue.
